### PR TITLE
[[ Bug 16278 ]] Add iOS amongst the supported platforms (with valid X…

### PR DESCRIPTION
**Pull request by livecodesebastien**
…code list)

### CODE REVIEW

File: docs/notes/platforms.md [hunk 1]
```diff
@@ -28,4 +28,24 @@ The Mac engine supports:
 *10.7.x (Lion) on Intel
 *10.8.x (Mountain Lion) on Intel
 *10.9.x (Mavericks) on Intel
+*10.10.x (Yosemite) on Intel
+*10.11.x (El Capitan) on Intel
 * **Note:** The engine runs as a 32-bit application regardless of the capabilities of the underlying processor.*
+## iOS
+iOS deployment is possible when running LiveCode IDE on a Mac, and provided Xcode is installed and has been set in LiveCode *Preferences* (in the *Mobile Support* pane).
+Currently, the supported versions of Xcode are:
+*Xcode 4.6 on MacOS X 10.6
```
Comment (runrevmark) [inline docs/notes/platforms.md]: Xcode 4.2 is the last to run on MacOS 10.6 - do we still build iOS engines that can work with that version?

Comment (livecodesebastien) [inline docs/notes/platforms.md]: Hm good point, in `revdeploylibraryios.livecodescript`, we simply take Xcode 4.6 for MacOS X < 10.8, but I didn't think that it was not the same for 10.7 and 10.6.
I'll remove the line for 10.6

**Comment by peter-b** [comment]
@livecode-vulcan review ok 59e3eaf

@livecodesebastien Don't forget to remove iOS 5.1 support when this is merged up to develop.

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 59e3eafafd1951ecf5910c1519e7994299377546

**Comment by livecode-vulcan** [comment]
😎 test success 59e3eafafd1951ecf5910c1519e7994299377546
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1194)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1205)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1191)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1198)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1209)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1211)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1213)

**Pull request merged**